test/prob/test_fixture_ccdf_log.hpp

```diff
@@ -273,7 +276,7 @@ class AgradCcdfLogTestFixture : public ::testing::Test {
   // works for fvar<double>
   double calculate_gradients_1storder(vector<double>& grad,
                                       fvar<double>& ccdf_log, vector<var>& x) {
-    x.push_back(ccdf_log.d_);
+    grad.push_back(ccdf_log.d_);
```

Comment:
Pushing stuff into x doesn't do anything. I think this was a bug. grad is the thing that gets checked.